packages/rest/src/http/types.ts

```diff
@@ -261,6 +267,7 @@ export interface HttpOperation {
 export interface RoutePath {
   path: string;
   isReset: boolean;
+  parameters?: Model;
```

Comment:
As said above and you pointed out, it would be better to have that be an interface. I think we could actually just add `shared?: boolean` at the root of this object as its already a container.

Reply:
I flattened this out as we discussed offline.